[2025-09-24T09:09:50.396+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_d3 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:09:50.406+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_d3 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:09:50.407+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T09:09:50.420+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): calculate_data_d3> on 2025-09-24 08:00:00+00:00
[2025-09-24T09:09:50.431+0000] {standard_task_runner.py:60} INFO - Started process 1954 to run task
[2025-09-24T09:09:50.434+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'calculate_data_d3', 'scheduled__2025-09-24T08:00:00+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmpcb0k2f0m']
[2025-09-24T09:09:50.436+0000] {standard_task_runner.py:88} INFO - Job 346: Subtask calculate_data_d3
[2025-09-24T09:09:50.495+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_d3 scheduled__2025-09-24T08:00:00+00:00 [running]> on host feaba2204362
[2025-09-24T09:09:50.582+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='calculate_data_d3' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-24T08:00:00+00:00'
[2025-09-24T09:09:50.728+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:09:50.751+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=calculate_data_d3, execution_date=20250924T080000, start_date=20250924T090950, end_date=20250924T090950
[2025-09-24T09:09:50.818+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:09:50.856+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-24T09:59:48.640+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_d3 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:59:48.650+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_d3 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:59:48.652+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T09:59:48.671+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): calculate_data_d3> on 2025-09-24 08:00:00+00:00
[2025-09-24T09:59:48.682+0000] {standard_task_runner.py:60} INFO - Started process 1316 to run task
[2025-09-24T09:59:48.687+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'calculate_data_d3', 'scheduled__2025-09-24T08:00:00+00:00', '--job-id', '404', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmpwsuonqko']
[2025-09-24T09:59:48.697+0000] {standard_task_runner.py:88} INFO - Job 404: Subtask calculate_data_d3
[2025-09-24T09:59:48.769+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_d3 scheduled__2025-09-24T08:00:00+00:00 [running]> on host 5d539c157d54
[2025-09-24T09:59:48.841+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='calculate_data_d3' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-24T08:00:00+00:00'
[2025-09-24T09:59:49.011+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:59:49.020+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=calculate_data_d3, execution_date=20250924T080000, start_date=20250924T095948, end_date=20250924T095949
[2025-09-24T09:59:49.079+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:59:49.147+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
